src/styles/card.scss

```diff
@@ -0,0 +1,126 @@
+@import 'variables';
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
```

Comment:
Avoid using fixed widths and heights for components. This can make the component less responsive and flexible. Consider using max-width or max-height for better responsiveness.